extensions/amp-analytics/0.1/events.js

```diff
@@ -419,20 +421,48 @@ export class TimerEventTracker extends EventTracker {
    */
   constructor(root) {
     super(root);
-    /** @const @private {!Array<number>} */
-    this.trackers_ = [];
+    /** @const @private {!Object<number, Object>} */
+    this.trackers_ = {};
+
+    /** @private {number} */
+    this.timerIdSequence_ = 1;
+
+    /**
+     * Timer event param names.
+     * @const
+     * @enum {number}
+     * @private
+     */
+    this.TIMER_PARAMS_ = {
```

Comment:
Here we are basically constructing a single timer event instance?  can we create an instance or typeDef instead? I honestly think a handler class would be best.